### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by BadScribbles** [QWERTxD/BetterDiscordPlugins#266]
**Describe the Bug**
Fuzzy search crashes discord when the Looked at fields are omitted `name, author, description, version`

Since Name is obligatory, that leaves us with the other three, BD assigns `Unknown Author` and `Description not provided.` to author and description fields respectively using `String` constructor which causes problems. 
*version field is supplied a string literal, so it's fine.*

So basically, if a plugin/theme is lacking a author or description, The search function of QuickToggler will crash discord.

---

**To Reproduce**
Add a plugin/theme with valid META except author and/or description field, and attempt to search anything
```js
/**
  * @name bug
  */

module.exports = class bug {
    start(){}
    stop(){}
}
```

---

**Expected Behavior**
Please don't crash my discord.

**Comment by Obazzi** [comment]
Haven't checked if one of my plugin/theme has empty author/description field but I can tell you it dose crash after entering a letter in the search.
